components/MKRInput.tsx

```diff
@@ -119,6 +123,11 @@ export function MKRInput({
           Please enter a valid amount.
         </Text>
       )}
+      {errorInvalidFormat && (
+        <Text sx={{ color: 'error', fontSize: 2 }} data-testid="mkr-input-error">
+          Please, enter a valid number.
```

Comment:
No `,` necessary here.

Reply:
Why, not? I , like, commas, thanks, see, you, later